addons/kibana/kibana.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta2
+kind: Addon
+metadata:
+  name: kibana/enterprise
+  labels:
+    kubeaddons.mesosphere.io/name: kibana/enterprise
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "6.8.10-11"
```

Comment:
The revision number should start with 1 which would result in "6.8.10-1" if this is initially added to this repo.